src/macros.jl

```diff
@@ -463,8 +463,8 @@ end
 
 
 function transform_helper(x, args...)
-    exprs = create_args_vector(args...)
-    t = (fun_to_vec(ex; gensym_names = false, nolhs = false) for ex in exprs)
+    exprs, wrap_ByRow = create_args_vector(args...)
+    t = [fun_to_vec(ex; gensym_names = false, nolhs = false, wrap_ByRow = wrap_ByRow) for ex in exprs]
```

Comment:
Why use `[` here but not in other functions?

Reply:
debugging, sorry.